configure.ac

```diff
@@ -811,7 +811,7 @@ fi
 AC_MSG_CHECKING([whether to turn sanitizers on])
 if test "x$with_sanitizer" != "x"; then
     AC_MSG_RESULT([yes ($with_sanitizer)])
-    SANITIZER_FLAGS="-O1 -fno-omit-frame-pointer -fsanitize=$with_sanitizer -frtti -static-libasan"
+    SANITIZER_FLAGS="-O1 -fno-omit-frame-pointer -fsanitize=$with_sanitizer -frtti"
```

Comment:
This is fine, my understanding is that the Jenkins fuzzer setup uses the sanitizer options from lode.git (set via a custom CC and CXX), so you can tweak this to your needs without problems.